### ISSUE TITLE
Add content type header for json arguments in documentation example

### ISSUE THREAD

**Issue by erikor** [rstudio/plumber#725]
### tl;dr: 

Change:

```
curl --data '{"a":4, "b":5}' http://localhost:8000/sum
```

To:

```
curl -H "Content-Type: application/json" --data '{"a":4, "b":5}' http://localhost:8000/sum
```

In the [plumber docs](https://www.rplumber.io/index.html).

### System details

```
R version 4.0.2 (2020-06-22)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Catalina 10.15.6

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] plumber_1.0.0.9999

loaded via a namespace (and not attached):
 [1] compiler_4.0.2      magrittr_2.0.1      R6_2.5.0            later_1.1.0.1      
 [5] promises_1.1.1.9001 tools_4.0.2         swagger_3.33.1      yaml_2.2.1         
 [9] Rcpp_1.0.5          stringi_1.5.3       jsonlite_1.7.2      webutils_1.1       
[13] httpuv_1.5.4        lifecycle_0.2.0     rlang_0.4.9.9000   

```

### Example application *or* steps to reproduce the problem

According to the [plumber docs](https://www.rplumber.io/index.html), I should be able to pass function arguments in JSON format. Given the following server:

```R
#* Return the sum of two numbers
#* @param a The first number to add
#* @param b The second number to add
#* @post /sum
function(a, b) {
  as.numeric(a) + as.numeric(b)
}
```

If I then start the server:

```R
library(plumber)
library(dplyr)
pr("plumber.R") %>%
     pr_run(port=8000)
```

This works as expected:

```
curl --data "a=4&b=3" "http://localhost:8000/sum"
[7]
```

But this does not work, although the docs linked above seem to state it should:

```
curl --data '{"a":4, "b":5}' http://localhost:8000/sum
```

The error I get is:

```
<simpleError in (function (a, b) {    as.numeric(a) + as.numeric(b)})(): argument "a" is missing, with no default>
```

### Solution

The solution is simply to declare the content type: 

```
curl  -H "Content-Type: application/json" --data '{"a":4, "b":5}' http://localhost:8000/sum
```
 Can this be clarified in the documentation reference above?

**Comment by meztez** [comment]
same as https://github.com/rstudio/plumber/issues/691
